app/editor/src/components/toggle-group/ToggleGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import * as styled from './styled';
+
+type IToggleOption = {
+  label: string;
+  onClick?: () => void;
+};
+
+export interface IToggleGroupProps {
+  options?: IToggleOption[];
```

Comment:
Would you ever use this without options?

Reply:
Nope, forgot to remove that. Was for the compiler at one point